src/xcode/ENA/ENA/Source/Developer Menu/DMSrsPrechecks/DMSrsPrechecksViewController.swift

```diff
@@ -0,0 +1,81 @@
+////
+// 🦠 Corona-Warn-App
+//
+#if !RELEASE
+
+import UIKit
+
+class DMSRSPrechecksViewController: UITableViewController {
+
+	// MARK: - Init
+
+	init(
+		store: Store
+	) {
+		self.viewModel = DMSRSPrechecksViewModel(store: store)
+
+		if #available(iOS 13.0, *) {
+			super.init(style: .insetGrouped)
+		} else {
+			super.init(style: .grouped)
+		}
+	}
+
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	// MARK: - Overrides
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		setupNavigationBar()
+		setupTableView()
+	}
+
+	// MARK: - Protocol UITableViewDataSource
+
+	override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		let cellViewModel = viewModel.cellViewModel(by: indexPath)
+		if cellViewModel is DMSwitchCellViewModel {
+			guard let cell = tableView.dequeueReusableCell(withIdentifier: DMSwitchTableViewCell.reuseIdentifier) as? DMSwitchTableViewCell else {
+				fatalError("unsupported cellViewModel - can't find a matching cell")
```

Comment:
Do we have an alternative way, instead of App Crash? Alert that something goes wrong or something? I know, it's the Dev Menu, so it's not so important (this remark)

Reply:
Good point, let's discuss this in our calls when Omar is back